showkase-processor/src/main/java/com/airbnb/android/showkase/processor/writer/WriterUtils.kt

```diff
@@ -342,6 +346,18 @@ internal fun generatePropertyNameFromMetadata(
     }
 }
 
+internal fun ShowkaseBrowserProperties.getPackageName() = zip()
+    .first()
+    .propertyPackage
+
+internal fun Collection<ShowkaseMetadata>.getNormalizedPackageName() = this
+    .first()
```

Comment:
is it safe to assume this is non null?

Reply:
I do a check before calling this but maybe I do this at the util level so that it's future proof.

Reply:
I agree it's better to be safe at the util level to not make assuumptions about where its called